### ISSUE TITLE
Unexpected setup data version: 6.1.0 (unicode)

### ISSUE THREAD

**Issue by ExaFlop1** [dscharrer/innoextract#143]
innoextract.exe -e "M:\Install\Sante.DICOM\Sante.DICOM.Viewer.3D.Pro\sante_dicom_viewer_pro.exe" -d "M:\Install\Sante.DICOM\Sante.DICOM.Viewer.3D.Pro\App" -m -s --collisions rename-all
Warning: Unexpected setup data version: 6.1.0 (unicode)
Warning: Unexpected trailing byte in UTF-16 string.
Warning: Unexpected data while converting from UTF-16LE to UTF-8.
Stream error while parsing setup headers!
 ├─ detected setup version: 6.1.0 (unicode)
 └─ error reason: basic_ios::clear: iostream error
If you are sure the setup file is not corrupted, consider
filing a bug report at https://innoextract.constexpr.org/issues
Done with 1 error and 3 warnings.

**Comment by ExaFlop1** [comment]
Again with another file - Driver Magician Lite
innoextract.exe -e "M:\Repair\Driver Magician\Lite\DML.exe" -d "M:\Repair\Driver Magician\Lite\temp" -m -s
Warning: Unexpected setup data version: 6.1.0 (unicode)
Warning: Unexpected trailing byte in UTF-16 string.
Warning: Unexpected data while converting from UTF-16LE to UTF-8.
Stream error while parsing setup headers!
 ├─ detected setup version: 6.1.0 (unicode)
 └─ error reason: basic_ios::clear: iostream error
If you are sure the setup file is not corrupted, consider
filing a bug report at https://innoextract.constexpr.org/issues
Done with 1 error and 3 warnings.
There is no problem with the file - InnoExtractor opening correctly.

**Comment by ExaFlop1** [comment]
And again same error:
`innoextract.exe -e "M:\OBDII\Creosys.OBD.Auto.Doctor\x64\obd-auto-doctor_4.3.2-l.exe" -d "M:\OBDII\Creosys.OBD.Auto.Doctor\x64" -m -s --collisions rename-all
Warning: Unexpected setup data version: 6.1.0 (unicode)
Warning: Unexpected trailing byte in UTF-16 string.
Warning: Unexpected data while converting from UTF-16LE to UTF-8.
Stream error while parsing setup headers!
 ├─ detected setup version: 6.1.0 (unicode)
 └─ error reason: basic_ios::clear: iostream error
If you are sure the setup file is not corrupted, consider
filing a bug report at https://innoextract.constexpr.org/issues
Done with 1 error and 3 warnings.`

**Comment by dscharrer** [comment]
Which version of innoextract are you using. You need at least innoextract 1.9 for those installers.

**Comment by ExaFlop1** [comment]
Yes, it is v.1.9.

**Comment by dscharrer** [comment]
I have downloaded the three installers
* https://www.drivermagician.com/DML.exe
* https://www.santesoft.eu/files/win/sante_dicom_viewer_pro.zip
* https://www.obdautodoctor.com/thankyou/?download=obd-auto-doctor_4.3.2-l.exe

And they all extract without errors for me. Perhaps the files you have are corrupt? Here are the checksums I get but it is possible that you have different versions - these are just the latest available online.

    137c6b2e2b55e18228241a1f30fd04d7  DML.exe
    8fea0d3631d38a68138cbc19fe0f328e  sante_dicom_viewer_pro.exe
    7feacf9a4fdb3262e5c1e5e64ba7e7ab  obd-auto-doctor_4.3.2-l.exe

**Comment by ExaFlop1** [comment]
No, they are not corrupt, because Havy's InnoExtractor opens them.

**Comment by dscharrer** [comment]
The please post the output of `innoectract.exe --version` as well as the checksums of the installers you are trying to extract.

**Comment by trivalik** [comment]
I can confirm all 3 works. But I found another one with exact same error https://www.scootersoftware.com/files/BCompare-5.0.0.29206.exe. Only happens with the beta version. Version 4 works properly.
